pkg/virt-handler/vm.go

```diff
@@ -656,6 +656,35 @@ func (d *VirtualMachineController) updateVMIStatus(vmi *v1.VirtualMachineInstanc
 		}
 	}
 
+	// Update AccessCredential conditions
+	if domain != nil && domain.Spec.Metadata.KubeVirt.AccessCredential != nil {
```

Comment:
Can we also have events sent? Especially of something goes wrong?

Reply:
yup, good idea.

Reply:
Great. Let's ensure that people can see without looking into logs that they get propagated or when it fails that they see in the events why it failed.

Reply:
done